indexing-service/src/main/java/io/druid/indexing/overlord/autoscaling/ec2/EC2AutoScaler.java

```diff
@@ -70,6 +72,13 @@ public EC2AutoScaler(
     this.envConfig = envConfig;
     this.amazonEC2Client = amazonEC2Client;
     this.config = config;
+
+    // Set the AWS region to work in according to envConfig, if set.
+    try {
+      amazonEC2Client.setRegion(RegionUtils.getRegion(envConfig.getRegion()));
+    } catch (Exception e) {
+      log.warn(String.format("Attempting to set an invalid region: %s", envConfig.getRegion()), e);
```

Comment:
mmx logger should take
`log.warn(e, "Attempting to set an invalid region: %s", envConfig.getRegion());`